backend/app.js

```diff
@@ -27,7 +27,7 @@ const resolvers = require('./graphql/resolvers')
 const typeDefs = require('./graphql/typeDefs')
 const schema = makeExecutableSchema({ typeDefs, resolvers })
 
-const graphQLEndpoint = '/graphql'
+const graphQLEndpoint = process.env.PUBLIC_URL?.includes('staging') ? '/luma-varaukset/graphql' : '/graphql'
```

Comment:
Näin saadaan graphql playground toimimaan oikein stagingilla. Tuotannossa pitäisi varmaan ottaa se sitten pois. Siihen voisi käyttää ehtoa 
```javascript
playground: (process.env.PUBLIC_URL?.includes('staging') || process.env.NODE_ENV ===  'development') ? 
{ 
  endpoint: graphQLEndpoint,
  subscriptionEndpoint: graphQLEndpoint + '/ws'
} : null
```